src/periphery/UpgradeHelper.sol

```diff
@@ -12,14 +12,15 @@ import {VaultStatusLib} from "../core/libs/VaultStatusLib.sol";
 contract UpgradeHelper {
     string public constant VERSION = "1.0.0";
```

Comment:
you need increase version of this contract because you changed method name platform -> PLATFORM